src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.html

```diff
@@ -305,42 +305,82 @@ <h5 jhiTranslate="artemisApp.exercise.export.options"></h5>
-                            >
-                                <a
-                                    [routerLink]="getAssessmentLink(value)"
-                                    class="btn btn-sm me-1"
-                                    [class.btn-success]="(!value.results?.[0]?.assessmentType || value.results?.[0].assessmentType === AssessmentType.AUTOMATIC) && !value.results?.[0]?.hasComplaint"
-                                    [class.btn-primary]="value.results?.[0]?.completionDate || value.results?.[0]?.hasComplaint"
-                                    [class.btn-warning]="!value.results?.[0]?.completionDate && !value.results?.[0]?.hasComplaint"
-                                >
-                                    <fa-icon [icon]="faFolderOpen" [fixedWidth]="true"></fa-icon>
-                                    <ng-container *ngIf="!value.results?.[0].hasComplaint; else courseComplaint">
-                                        {{ ('artemisApp.assessment.dashboard.actions.' +
-                                            (!value.results?.[0]?.assessmentType || value.results?.[0].assessmentType === AssessmentType.AUTOMATIC ? 'assess' : (value.results?.[0]?.completionDate ? 'open' : 'continue')))
-                                            | artemisTranslate
-                                        }}
-                                    </ng-container>
-                                    <ng-template #courseComplaint>
-                                        {{ 'artemisApp.exerciseAssessmentDashboard.showComplaint' | artemisTranslate }}
-                                    </ng-template>
-                                </a>
-                                <button
-                                    *ngIf="
+                            <ng-container *ngIf="correctionRoundIndices?.length && correctionRoundIndices.length > 1; else singleCorrectionRound">
+                                <ng-container *ngFor="let correctionRound of correctionRoundIndices">
+                                    <div
+                                        *ngIf="(correctionRound == 0 || value.results?.[correctionRound - 1]?.completionDate) &&
+                                        (newManualResultAllowed || (value.results?.[correctionRound]?.assessmentType && value.results[correctionRound].assessmentType !== AssessmentType.AUTOMATIC))"
+                                    >
+                                        <a
+                                            [routerLink]="getAssessmentLink(value, correctionRound)"
+                                            [queryParams]="{ 'correction-round': correctionRound }"
+                                            class="btn btn-sm me-1 mb-2"
+                                            [class.btn-success]="(!value.results?.[correctionRound]?.assessmentType || value.results?.[correctionRound].assessmentType === AssessmentType.AUTOMATIC) && !value.results?.[0]?.hasComplaint"
```

Comment:
```suggestion
                                            [class.btn-success]="(!value.results?.[correctionRound]?.assessmentType || value.results?.[correctionRound].assessmentType === AssessmentType.AUTOMATIC) && !value.results?.[correctionRound]?.hasComplaint"
```

Did you forget that one?